README.md

```diff
@@ -374,13 +376,43 @@ let app =
         ]
 ```
 
+### subRoute
+
+`subRoute` checks the current path begins with the `path` string, all `route*` and `subRoute*` handlers within the `handler` parameter will now be rooted to, and relative with respect to this path.
+
```

Comment:
Hmm... to be honest I have no preference for the parameter name. Can stick with `path` or pick which one you think makes more sense.

One last thing also re example. After removing the `>>=` does the `choose [ ... ]` need to be wrapped in `()` or will it work like this?